srtcore/core.cpp

```diff
@@ -993,6 +995,11 @@ void CUDT::setOpt(SRT_SOCKOPT optName, const void* optval, int optlen)
             m_uOPT_StabilityTimeout = val * 1000;
         }
 
+    case SRTO_RETRANSMISSION_ALGORITHM:
+        if (m_bConnected)
+            throw CUDTException(MJ_NOTSUP, MN_ISCONNECTED, 0);
+
+        m_iOPT_RexmitAlgo = cast_optval<int>(optval, optlen);
```

Comment:
Please use fixed size integer always for the options (a pending PR is there to fix them all). Here, `int32_t`.